docs/additional-resources/blue-green-migration.md

```diff
@@ -30,12 +68,17 @@ Some key steps, excluding automated tests, include:
 - Delete the source's DynamoDB table in `east` and `west` regions.
 - Toggle the deploy table's values for `migrate` and `source-table-version`.
 
-
 ## Manual Migration Steps
 
 The application kicks off a migration automatically if it detects migrations that need to be run in the codebase that haven't yet been run upon that environment. In order to force a migration, perform the following manual steps. You might do this if you were doing a package update that might impact the migration and wanted to test it fully.
 
-1. Remove a migration script item from the environment's `efcms-<ENV>-deploy` DynamoDB table that is expected to migrate, from the `migrationsToRun.js` file. For example, if `migrationsToRun.js` includes a script called `0001-test-script.js`, delete that item from the DynamoDB table.
-2. Kick off a CircleCI workflow, which will follow the steps above for Automated Migration Steps.
+1. Go into the environment's deploy table `efcms-<ENV>-deploy`
+2. Change the `migrate` flag to `true`
+3. Identify the source table and destination table. Both should be the same value, and that is what the application is currently using as it its main data store. If they are both `alpha`, then the destination table is `beta`. If they are both `beta`, the destination table is `alpha`. If they are different, inspect at the current color Lamdba's environment config to determine the dynamo table that the application is currently using as its data store.
+4. Switch to the `west-1` region.
```

Comment:
Nit: maybe explain here why to delete from west first

Reply:
Cando.